doc/node.1

```diff
@@ -1,368 +1,352 @@
-environment variables.
-
-The bundled CA store, as supplied by Node.js, is a snapshot of Mozilla CA store
-that is fixed at release time. It is identical on all supported platforms.
-
-See \fBSSL_CERT_DIR\fR and \fBSSL_CERT_FILE\fR.
-
-.TP
-.BR \-\-icu\-data\-dir =\fIfile\fR
-Specify ICU data load path. (overrides \fBNODE_ICU_DATA\fR)
-
-.TP
-.BR \-\fR
-Alias for stdin, analogous to the use of - in other command line utilities,
-meaning that the script will be read from stdin, and the rest of the options
-are passed to that script.
-
-.TP
-.BR \-\-\fR
-Indicate the end of node options. Pass the rest of the arguments to the script.
-
+.
+.It Fl -prof-process
+Process V8 profiler output generated using the V8 option
+.Fl -prof .
+.
+.It Fl -v8-options
+Print V8 command-line options.
+.Pp
+Note: V8 options allow words to be separated by both dashes (\fB-\fR) or underscores (\fB_\fR).
+.Pp
+For example,
+.Fl -stack-trace-limit
+is equivalent to
+.Fl -stack_trace_limit .
+.
+.It Fl -v8-pool-size Ar num
+Set V8's thread pool size which will be used to allocate background jobs.
+If set to 0 then V8 will choose an appropriate size of the thread pool based on the number of online processors.
+If the value provided is larger than V8's maximum, then the largest value will be chosen.
+.
+.It Fl -tls-cipher-list Ar list
+Specify an alternative default TLS cipher list.
+Requires Node.js to be built with crypto support. (Default)
+.
+.It Fl -enable-fips
+Enable FIPS-compliant crypto at startup.
+Requires Node.js to be built with
+.Sy ./configure --openssl-fips .
+.
+.It Fl -force-fips
+Force FIPS-compliant crypto on startup
+(Cannot be disabled from script code).
+Same requirements as
+.Fl -enable-fips .
+.
+.It Fl -openssl-config Ar file
+Load an OpenSSL configuration file on startup.
+Among other uses, this can be used to enable FIPS-compliant crypto if Node.js is built with
+.Sy ./configure --openssl-fips .
+.
+.It Fl -use-openssl-ca , Fl -use-bundled\-ca
+Use OpenSSL's default CA store or use bundled Mozilla CA store as supplied by current Node.js version.
+The default store is selectable at build-time.
+.Pp
+Using OpenSSL store allows for external modifications of the store.
+For most Linux and BSD distributions, this store is maintained by the distribution maintainers and system administrators.
+OpenSSL CA store location is dependent on configuration of the OpenSSL library but this can be altered at runtime using environment variables.
+.Pp
+The bundled CA store, as supplied by Node.js, is a snapshot of Mozilla CA store that is fixed at release time.
+It is identical on all supported platforms.
+.Pp
+See
+.Ev SSL_CERT_DIR
+and
+.Ev SSL_CERT_FILE .
+.
+.It Fl -icu-data-dir Ar file
+Specify ICU data load path.
+Overrides
+.Ev NODE_ICU_DATA .
+.
+.It Fl \-experimental-modules
+Enable experimental ES module support and caching modules.
+.
+.It Fl \-experimental-vm-modules
+Enable experimental ES module support in VM module.
+.
+.It \-
```

Comment:
Shouldn't this `-` be highlighted as a flag?

Reply:
It's not a real flag, as such. Both `-` and `--` probably shouldn't even be documented here. I'd move them back up to the `DESCRIPTION` block, next to the line about no arguments enabling the REPL.

Reply:
Sounds fine to me, they aren't really options, but I see some value having them documented.

Reply:
OTOH, I just noticed that for example `git` seems to also list `--` under options, see `git-commit(1)`. So maybe it's not that uncommon to list `--` and `-` as options.

Reply:
If it is going to stay here (which seems fine to me, especially given the `git-commit` example) then I guess bolding the `-` and `--` makes sense.